docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -282,6 +282,16 @@ The GeoJSON layer allows you to select and load a static GeoJSON file from the f
 - **Add style rule** creates additional style rules.
 - **Display tooltip** allows you to toggle tooltips for the layer.
 
+{{% admonition type="note" %}}
```

Comment:
I think we're no longer using Hugo `admonition` short codes in favor of `> **Note:** <you're note here>` syntax. @imatwawana can you confirm?

Reply:
We favour admonition shortcodes, but in the `transformations.md` file we can't use them because of how it's generated, so that might be what's causing confusion here.